src/mate/src/Bridge/Profiler/Capability/ProfilerResourceTemplate.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Mate\Bridge\Profiler\Capability;
+
+use Mcp\Capability\Attribute\McpResourceTemplate;
+use Symfony\AI\Mate\Bridge\Profiler\Service\ProfilerDataProvider;
+
+/**
+ * MCP resource templates for accessing Symfony profiler data.
+ *
+ * @author Johannes Wachter <johannes@sulu.io>
+ */
+final class ProfilerResourceTemplate
+{
+    public function __construct(
+        private ProfilerDataProvider $dataProvider,
+    ) {
+    }
+
+    /**
+     * @return array{uri: string, mimeType: string, text: string}
+     */
+    #[McpResourceTemplate(
+        uriTemplate: 'profiler://profile/{token}/metadata',
+        name: 'Profile Metadata',
+        description: 'Access profile metadata (method, URL, status, time, IP)'
+    )]
+    public function getProfileMetadataResource(string $token): array
+    {
+        $profile = $this->dataProvider->findProfile($token);
+
+        if (null === $profile) {
+            return [
+                'uri' => "profiler://profile/{$token}/metadata",
+                'mimeType' => 'application/json',
+                'text' => json_encode(['error' => 'Profile not found', 'token' => $token]) ?: '{}',
+            ];
+        }
+
+        $data = [
+            'token' => $profile->token,
+            'method' => $profile->index->method,
+            'url' => $profile->index->url,
+            'status_code' => $profile->index->statusCode,
+            'time' => $profile->index->time,
+            'time_formatted' => date(\DateTimeInterface::ATOM, $profile->index->time),
+            'ip' => $profile->index->ip,
+            'parent_token' => $profile->index->parentToken,
+        ];
+
+        if (null !== $profile->index->context) {
+            $data['context'] = $profile->index->context;
+        }
+
+        return [
+            'uri' => "profiler://profile/{$token}/metadata",
+            'mimeType' => 'application/json',
+            'text' => json_encode($data, \JSON_PRETTY_PRINT) ?: '{}',
+        ];
+    }
+
+    /**
+     * @return array{uri: string, mimeType: string, text: string}
+     */
+    #[McpResourceTemplate(
+        uriTemplate: 'profiler://profile/{token}/request',
```

Comment:
maybe?
```suggestion
        uriTemplate: 'profiler://profile/{token}/{collector}',
```

would also enable 3rd party collector